src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -29,7 +29,7 @@
         "symfony/password-hasher": "^5.4|^6.0|^7.0",
         "symfony/security-core": "^6.2|^7.0",
         "symfony/security-csrf": "^5.4|^6.0|^7.0",
-        "symfony/security-http": "^6.3|^7.0"
+        "symfony/security-http": "^6.3.4|^7.0"
```

Comment:
Related to issue https://github.com/symfony/symfony/pull/51104

result before. https://github.com/symfony/symfony/actions/runs/6138815527/job/16656061481#step:8:1778

Reply:
This change should go to the 6.3 branch then.

Reply:
Done, moved to PR https://github.com/symfony/symfony/pull/51627

Reply:
Merged. Can you please rebase this one?

Reply:
Done